src/components/popup-menu/popup-menu.styles.js

```diff
@@ -0,0 +1,111 @@
+export default ({ palette, typography, mixins }) => ({
```

Comment:
Does this support changing the font-size of the menu?

Reply:
Nope.